Frontend/library/src/PeerConnectionController/PeerConnectionController.ts

```diff
@@ -407,6 +407,26 @@ export class PeerConnectionController {
                             });
                         });
 
+                    // If the user has toggle the "UseFlexFec" flag, we want to add it to the top of the preferences list.
+                    // We don't want it as a selectable codec because the first "actual" codec in the list is the codec that will be
+                    // protected.
+                    if(this.config.isFlagEnabled(Flags.UseFlexFec)) {
+                        const matcher = /(flexfec).*/;
```

Comment:
To save ourselves having to write something similar when we want to add ulpfec of RED support, can we add all the capabilities to our codec preferences?